website/docs/index.html.markdown

```diff
@@ -43,7 +43,7 @@ The following arguments are supported:
 * `enable_ddi` - (Optional) This sets the permission schema to a DDI-compatible schema. 
 Users of the managed SaaS product should not need to set this.
 Users of DDI should set this to true if managing teams, users, or API keys through this provider.
-* `rate_limit_parallelism` - (Optional) Integer for parallelism amount (terraform's default is 10).
+* `rate_limit_parallelism` - (Optional) Integer for alternative rate limit and parallelism strategy. (Terraform default value is 10.)
```

Comment:
Typo?  Period should be outside of parenthetical

Reply:
ugh. moving too quickly. good catch. fixing now. sorry!

Reply:
oh wait not that's correct. The whole sentence is inside the parenthesis.